docs/migration/2211_ng19/migration.md

```diff
@@ -17,22 +17,22 @@ git commit -m "update angular 18 and 3rd party deps angular 18 compatible"
 ```
 Note: Do not select `use-application-builder` migration when migrating to Angular 18. Applications created before SPA 2211.19 doesn't support this builder. Applications created starting from 2211.19 already supports it.
 
-### Run Spartacus update
+### Update Angular to 19 and 3rd party deps to be compatible with Angular 19
 
-After successfully updating the application to Angular 18, execute this command to initiate the Spartacus update process.
+Follow the [Angular guidelines for upgrading from v18 to v19](https://angular.dev/update-guide?v=18.0-19.0&l=3) and bump the Angular version locally, and update other 3rd party dependencies from Angular ecosystem  to versions compatible with Angular 19 (e.g. `@ng-select/ng-select@14`, `@ngrx/store@19`, `ngx-infinite-scroll@19`):
 
 ```bash
-ng update @spartacus/schematics@latest
+ng update @angular/cli@19 @angular/core@19 ngx-infinite-scroll@19 @ng-select/ng-select@14 @ngrx/store@19 angular-oauth2-oidc@19 --force
+git add .
+git commit -m "update angular 19 and 3rd party deps angular 19 compatible"
 ```
 
-### Adjust Angular configuration
+Note: Unselect `use-application-builder` migration when migrating to Angular 19. Applications created before SPA 2211.19 doesn't support this builder. Applications created starting from 2211.19 already supports it.
```

Comment:
[MEDIUM]
This is said too late. People had already run the command. 
Instead let's put a bold warning _before running the command_ (in line 23). Also I'd mention that you need to hit a SPACE button to unselect this option, and only then enter.

Because it's easy to miss the small prompt during the installation, perhaps we might even attach 2 screenshots:- default preselected option vs unselected option with a space